cdap-api/src/main/java/co/cask/cdap/api/flow/flowlet/FlowletSpecification.java

```diff
@@ -76,6 +79,21 @@
   Resources getResources();
 
   /**
+   * @return Map of streams and {@link StreamSpecification} created in this flowlet
+   */
+  Map<String, StreamSpecification> getStreams();
```

Comment:
Why get repeat again? If these methods are needed in the specification, shall we have a super-interface for these methods?